squad/frontend/group_settings.py

```diff
@@ -31,6 +32,28 @@ def settings(request, group_slug):
     return render(request, 'squad/group_settings/index.jinja2', context)
 
 
+class NewGroupForm(GroupForm):
+    class Meta(GroupForm.Meta):
+        fields = ['slug'] + GroupForm.Meta.fields
+
+
+@login_required
+def new_group(request):
+    if request.method == "POST":
+        form = NewGroupForm(request.POST, instance=Group())
+        if form.is_valid():
+            group = form.save()
+            group.setup_for(request.user)
+            return redirect(reverse('group-settings', args=[group.slug]))
```

Comment:
Same as with 'new project'. In this case there is no crash but there is no error message either.